<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1255</width>
    <height>855</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_title">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>361</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monotype Corsiva</family>
      <pointsize>30</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Firmă de catering</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_fel_principal">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Fel principal</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_bauturi">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Băuturi</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_desert">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>80</y>
      <width>411</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Desert</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>120</y>
      <width>770</width>
      <height>401</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Produs</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Pret</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ingrediente</string>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>221</width>
      <height>401</height>
     </rect>
    </property>
   </widget>
   <widget class="QListWidget" name="list_view_selectii">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>580</y>
      <width>741</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_adauga">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>170</y>
      <width>161</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Adauga selectia in cos</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_selectii">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Selectiile tale</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_plaseaza_cmd">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>640</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Plaseaza comanda</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_total_plata">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>590</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Total de platit:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_succes">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>690</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Comanda plasata cu succes</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1255</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
